### ISSUE TITLE
Get-IStat missing from PowerForensics

### ISSUE THREAD

**Issue by jahshuah** [Invoke-IR/PowerForensics#1]
I'll admit, I'm probably missing something, but when I download PowerForensics and import the DLL, Get-IStat is missing (as is Get-ICat).  Are these cmdlets included with the PowerForensics master.zip download?  Thanks!

**Comment by Invoke-IR** [comment]
Hi jahshuah,

You are not incorrect.  I realized that the names Get-IStat and Get-ICat
required an understanding of The Sleuth Kit to completely understand their
functionality, and decided that I did not want that dependency.  I
combined/renamed the cmdlets in your question to Get-MFTRecord.  To do ICat
functionality you must specify the -AsBytes switch parameter.

I will update the repo with details to properly install the module tonight.

Thanks,
Jared
